tests/integration/raw_studies_blueprint/test_fetch_raw_data.py

```diff
@@ -297,206 +298,405 @@ def test_get_study(self, client: TestClient, user_access_token: str, internal_st
-
-    # we can't create a folder and specify a content at the same time
-    res = client.put(raw_url, params={"path": "", "resource_type": "folder"}, files={"file": b"content"})
-    assert res.status_code == 422
-    assert res.json()["description"] == "Argument mismatch: Cannot give a content to create a folder"
-
-    # try to create a folder outside `user` folder
-    wrong_folder = "input/wrong_folder"
-    expected_msg = f"the given path isn't inside the 'User' folder: {wrong_folder}"
-    res = client.put(raw_url, params={"path": wrong_folder, **additional_params})
-    assert res.status_code == 403
-    assert res.json()["exception"] == "FolderCreationNotAllowed"
-    assert expected_msg in res.json()["description"]
-
-    # try to create a folder inside the 'expansion` folder
-    expansion_folder = "user/expansion/wrong_folder"
-    expected_msg = "you are not allowed to create a resource here"
-    res = client.put(raw_url, params={"path": expansion_folder, **additional_params})
-    _check_endpoint_response(study_type, res, client, internal_study_id, expected_msg, "FolderCreationNotAllowed")
-
-    # try to create an already existing folder
-    existing_folder = "user/folder_1"
-    expected_msg = "the given resource already exists"
-    res = client.put(raw_url, params={"path": existing_folder, **additional_params})
-    _check_endpoint_response(study_type, res, client, internal_study_id, expected_msg, "FolderCreationNotAllowed")
-
-
-def test_retrieve_from_archive(client: TestClient, user_access_token: str) -> None:
-    # client headers
-    client.headers = {"Authorization": f"Bearer {user_access_token}"}
-
-    # create a new study
-    res = client.post("/v1/studies?name=MyStudy")
-    assert res.status_code == 201
-
-    # get the study id
-    study_id = res.json()
-
-    # add a new area to the study
-    res = client.post(
-        f"/v1/studies/{study_id}/areas",
-        json={
-            "name": "area 1",
-            "type": "AREA",
-            "metadata": {"country": "FR", "tags": ["a"]},
-        },
-    )
-    assert res.status_code == 200, res.json()
-
-    # archive the study
-    res = client.put(f"/v1/studies/{study_id}/archive")
-    assert res.status_code == 200
-    task_id = res.json()
-    wait_for(
-        lambda: client.get(
-            f"/v1/tasks/{task_id}",
-        ).json()["status"]
-        == 3
-    )
-
-    # retrieve a `Desktop.ini` file from inside the archive
-    rel_path = "Desktop"
-    res = client.get(
-        f"/v1/studies/{study_id}/raw",
-        params={"path": rel_path, "formatted": True},
-    )
-    assert res.status_code == 200
-
-    # retrieve a `study.antares` file from inside the archive
-    rel_path = "study"
-    res = client.get(
-        f"/v1/studies/{study_id}/raw",
-        params={"path": rel_path, "formatted": True},
-    )
-    assert res.status_code == 200
+        res.raise_for_status()
+
+        # Create a cluster in the first area
```

Comment:
You don't need to create the cluster and the area, it's just misleading as it doesn't bring anything. You can just create the BC like this and it still works: 
```python
bc_obj = {
    "name": "bc_1",
    "enabled": True,
    "time_step": "daily",
    "operator": "less",
}
res = client.post(f"/v1/studies/{study_id}/bindingconstraints", json=bc_obj)
res.raise_for_status()
```